index.html

```diff
@@ -3,314 +3,423 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 2: Javascript and the DOM</title><!-- Latest compiled and minified CSS -->
+  <title>Felony Disenfranchisement</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Fjalla+One|Exo:400,600|Ropa+Sans' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=PT+Sans+Narrow|Arimo|Dosis|Lato' rel='stylesheet' type='text/css'>
+
   <style>
     .section { padding: 1em 0; }
-    /* An "em" is equal to the height of a line of text. */
     .header {
-      background: linear-gradient(to bottom, #337ab7 0%, #135a97 100%);
-      color: white;
-      padding: 1em 0 2em;
+      text-align: left;
+    }
+    table {
+      border-collapse: collapse;
+      width: 100px;
+    }
+
+    th, td {
+      padding: 8px;
+      text-align: left;
+      border-bottom: 1px solid #ddd;
     }
     #data-table td {
-      width: 25%;
-      /* The table has four columns, so this rule makes them of equal width. */
+      width: 20%;
+      /* The table has 5 columns, so this rule makes them of equal width. */
```

Comment:
Great updating of the comment. I've confused myself many times with lying comments that were never updated.